examples/source/interactivity-dynamic-content/Star_Rating/index.html

```diff
@@ -129,7 +129,7 @@
   We want the form to submit as soon as the user makes a selection, without a Submit button. To do that, we'll set the [`on` attribute](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#on) of the `input`s to submit the form on [`change`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-form.md', locale=doc.locale).url.path}}#input-events).
 
   The initial rating is determined by which radio button has the `checked` attribute set. This is optional. -->
-  <form id="rating" class="p2" method="post" action-xhr="<% hosts.backend %>/samples_templates/rating/set" target="_blank">
+  <form id="rating" class="p2" method="post" action-xhr="set" target="_blank">
```

Comment:
Can you please remove `class="p2"` and there's also a ` class="p1"` somewhere else in the file. It's an ampstart leftover we no longer need. Thanks!

Reply:
Sure. That whole paragraph that had the `class="p1"` seems to be outdated anyways.

Reply:
Why does it seem outdated?

Reply:
Sorry, I was confused for a bit, because that snippet was not showing up, but now everything seems fine.